test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -400,21 +400,33 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $result.Error | Should BeNullOrEmpty
     }
 
-    It "Validate StandardMethod and CustomMethod parameter sets" {
+    It "Validate Invoke-WebRequest StandardMethod and CustomMethod parameter sets" {
         
         #Validate that parameter sets are functioning correctly
-        $command = "Invoke-RestMethod -Uri 'http://sandbox.lee.io/method.php' -Method GET -CustomMethod 'TEST'"
-        $result = ExecuteWebCommand -command $command
-        $result.Error | Should Not BeNullOrEmpty
+        $errorId = "AmbiguousParameterSet,Microsoft.PowerShell.Commands.InvokeWebRequestCommand"
+        { Invoke-WebRequest -Uri 'http://http.lee.io/method' -Method GET -CustomMethod TEST } | ShouldBeErrorId $errorId
     }
 
-    It "Validate CustomMethod parameter set method is passed" {
+    It "Validate Invoke-WebRequest CustomMethod method is used" {
         
-        #Validate that custom method is used
-        $command = "Invoke-RestMethod -Uri 'http://http.lee.io/method' -CustomMethod 'TEST'"
+        $command = "Invoke-WebRequest -Uri 'http://http.lee.io/method' -CustomMethod TEST"
         $result = ExecuteWebCommand -command $command
         $result.Error | Should BeNullOrEmpty
-        $result.output.method | Should Be "TEST"
+        ($result.Output.Content | ConvertFrom-Json).output.method | Should Be "TEST"
+    }
+
+    It "Validate Invoke-WebRequest default ContentType for CustomMethod POST" {
+        
+        $command = "Invoke-WebRequest -Uri 'http://httpbin.org/post' -CustomMethod POST -Body 'testparam=testvalue'"
+        $result = ExecuteWebCommand -command $command
+        ($result.Output.Content | ConvertFrom-Json).headers.'Content-Type' | Should Be "application/x-www-form-urlencoded"
```

Comment:
since you're sending a specific body, seems like you should just check it was returned correctly in the response:

$result.Output.form.testvalue | Should Be "testvalue"